README.md

```diff
@@ -46,32 +46,33 @@ Values that can be configured by using environment variables:
+| `SYMBOLS` | No | The symbols that the oracle component collects data points for | "AUD-USD,CAD-USD,EUR-USD,GBP-USD,JPY-USD,SEK-USD,ATN-USD,NTN-USD,NTN-ATN"                                    | symbols separated by ',' |
+| `PLUGIN.DIR` | No | The directory that stores the plugins | "./build/bin/plugins"                                                                                | any directory that saves plugins |
+| `KEY.FILE` | Yes | The encrypted key file path that contains the private key of the oracle client. | "./test_data/keystore/UTC--2023-02-27T09-10-19.592765887Z--b749d3d83376276ab4ddef2d9300fb5ce70ebafe" | any key file that saves the private key |
+| `KEY.PASSWORD` | Yes | The password of the key file that contains the private key of the oracle client. | "123"                                                                                                | any password that encrypted the private key |
+| `AUTONITY.WS.URL` | Yes | The web socket RPC URL of your Autonity L1 Node that the oracle client communicates with. | "ws://127.0.0.1:8546"                                                                                | the web socket rpc endpoint url of the Autonity client. |
+| `PLUGIN.CONF` | Yes | The plugins' configuration file in YAML. | "./build/bin/plugins/plugins-conf.yml"                                                               | the configuration file of the oracle plugins. |
+| `GAS.TIP.CAP` | No | The gas priority fee cap to issue the oracle data report transactions | 1                                                               | A non-zero value per gas to prioritize your data report TX to be mined. |
+| `LOG.LEVEL` | No | The logging level of the oracle server | 2                                                              | available levels are:  0: NoLevel, 1: Trace, 2:Debug, 3: Info, 4: Warn, 5: Error. |
 
 
 or by using console flags:
 ```shell
 $./autoracle -help  
 Usage of ./autoracle:
-  -oracle_autonity_ws_url="ws://127.0.0.1:8546": WS-RPC server listening interface and port of the connected Autonity Go Client node
-  -oracle_gas_tip_cap=1: The gas priority fee cap set for oracle data report transactions
-  -oracle_key_file="./test_data/keystore/UTC--2023-02-27T09-10-19.592765887Z--b749d3d83376276ab4ddef2d9300fb5ce70ebafe": Oracle server key file
-  -oracle_key_password="123": Password to the oracle server key file
-  -oracle_plugin_conf="./build/bin/plugins/plugins-conf.yml": The plugins' configuration file in YAML
-  -oracle_plugin_dir="./build/bin/plugins": The DIR where the adapter plugins are stored
-  -oracle_symbols="AUD-USD,CAD-USD,EUR-USD,GBP-USD,JPY-USD,SEK-USD,ATN-USD,NTN-USD,NTN-ATN": The currency pair symbols the oracle returns data for. A comma-separated list
+  -autonity.ws.url="ws://127.0.0.1:8546": WS-RPC server listening interface and port of the connected Autonity Go Client node
+  -gas.tip.cap=1: The gas priority fee cap set for oracle data report transactions
```

Comment:
Is this the cap or the actual tip set for the transaction?

Reply:
> Is this the cap or the actual tip set for the transaction?

It is the gas priority fee cap to use for the 1559 transaction execution, the higher value will let the TX get higher priority to be mined in a block.

Reply:
ok was wondering if we could shorten it to `--tip` for example.

Reply:
> ok was wondering if we could shorten it to `--tip` for example.

resolved via 955f5308bedc0599573d98eb992c8ef93d7e093a